index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <title>My Portofolio</title>
+</head>
+<body>
+
+  <div class="main-content">
```

Comment:
- [ ] It would be better to wrap this entire headline section of your code with a `<section>` tag instead of a `<div>`. The idea is to use semantic HTML and ensure as much as possible that your HTML code accurately describes your page structure 😎 
- [ ] Please add a background image to this section as specified in the Figma design
- [ ] Also ensure to align the content of this section with `flexbox`
- [ ] Please use margins as specified in the design